windows_registry/README.md

```diff
@@ -0,0 +1,67 @@
+# Windows Registry Integration (Beta)
+
+## Overview
+
+Watch for changes in Windows Registry keys and forward them to Datadog. Enable this integration to:
+
+- Understand system and application health through Windows Registry key values
+- Monitor for unexpected changes impacting security and compliance requirements
+
+## Setup
+
+### Installation
+
+The Windows Registry integration is included in the [Datadog Agent][1] package. No additional installation is needed.
+
+### Configuration
+
+This integration collects Windows Registry information using both of following methods:
+
+- As [Datadog Metrics][2]
+- As [Datadog Logs][3]
+
+
+1. Edit the `windows_registry.d/conf.yaml` file in the `conf.d/` folder at the root of your Agent's [configuration directory][4] to start collecting Windows registry information. See the [sample windows_.d/conf.yaml][5] for all available configuration options.
```

Comment:
should be `sample windows_.d` be `sample windows_registry.d`?

Reply:
yes good catch! must have accidentally edited it